libpdf/models/horizontal_box.py

```diff
@@ -17,6 +17,8 @@ class Char:  # pylint: disable=too-few-public-methods # simplicity is good.
     :vartype x1: float
     :ivar y1: distance from the bottom of the page to the upper edge of the character (greater than y0)
     :vartype y1: float
+    :ivar ncolor: non-stroking-color
+    :vartype ncolor: tuple of rgb or None
```

Comment:
currently the value taken is from PDF as rgb as floats. Should the value stored be converted to some other type (e.g. tuple of int in range [0,255] or to hex value? This could make it easier for comparisons.

Reply:
For the number format I have no strong preference. Off these options I'd go with the PDF standard / floats. [Matplotlib](https://matplotlib.org/stable/users/explain/colors/colors.html) also uses it as numeric input.
Converting the colors by any calling library should be easy. Floats could have a higher accuracy than the 8 bits we get from 0.255 for each color. I guess there are people that go crazy about color depth and accuracy.

Reply:
I see you went with float.